honeybee_radiance_command/gensky.py

```diff
@@ -207,8 +207,10 @@ def from_ang(cls, angles, options=None):
             options: Command options. It will be set to Radiance default values if not
                 provided by user.
         """
+        cls_from_ang = cls()
+
         if not options:
-            options = GenskyOptions()
+            options = cls_from_ang.options
 
         if angles:
```

Comment:
1. angles is required. You shouldn't have a check like this.
2. in line 218 you can simply return `cls_from_ang` instead of creating a new instance.